library/Network/IPFS/Add/Error.hs

```diff
@@ -20,4 +21,5 @@ instance Display Error where
     InvalidFile          -> "Invalid file"
     UnexpectedOutput txt -> "Unexpected IPFS output: " <> display txt
     RecursiveAddErr  err -> "Error while adding directory" <> display err
+    IPFSDaemonErr    txt -> "IPFS Daemon error:" <> display txt -- TODO: Move this type to it's own file
```

Comment:
Hmm, not sure I follow the `TODO` 🤔 Explain more?

Reply:
Oh, also probably want a space after `error:"`

Reply:
Oh it was because its generally an IPFS error not an error when adding/pinning. SO I was thinking on friday where this might better live. 

Though thinking about it now its still fine being here till we have a need to break it out further.